packages/ERTP/src/issuer.js

```diff
@@ -385,6 +385,13 @@ function produceIssuer(allegedName, mathHelpersName = 'nat') {
         return paymentLedger.get(payment);
       });
     },
+
```

Comment:
It's only necessary to emulate the old API on the new one, until we move all old contracts onto the new API. That's why it's marked "TODO(msm): remove".

It might be possible to do this emulation without temporarily adding this to issuer.js. But since it's just a transient...

Reply:
Supporting exactly the old API caused the CI integration test to pass, so resolving this as to-be-removed in the next PR.